source/develop/release-management/features/storage/vm-leases.html.md

```diff
@@ -189,39 +182,72 @@ using a VM lease on another host.
 
 This verb is implemented since ovirt-3.5 as internal verb, we will make
 this API public in 4.1.
```

Comment:
This can be used to check the status of a vm on a non-responsive host, but virt is not going to use this, they plan to start the vm on another host, and retrying if the vm fail to start, depending on the lease.